modules/rules_apple_linker/0.3.0/MODULE.bazel

```diff
@@ -0,0 +1,16 @@
+module(
+    name = "rules_apple_linker",
+    version = "0.3.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.3.0", dev_dependency = True)
+bazel_dep(name = "rules_apple", version = "2.0.0", dev_dependency = True, repo_name = "build_bazel_rules_apple")
+bazel_dep(name = "rules_swift", version = "1.5.0", dev_dependency = True, repo_name = "build_bazel_rules_swift")
```

Comment:
```suggestion
bazel_dep(name = "rules_swift", version = "1.5.1", dev_dependency = True, repo_name = "build_bazel_rules_swift")
```

Reply:
do i have to patch that in since it doesn't match my repo tho?

Reply:
hrm does it matter since it's only a dev dependency?

Reply:
There is a presubmit change checking that the module file in the registry matches the one in the source archive, but that seems to be the case here. This is meant to allow for a future vendor mode.

Dev dependencies don't matter for users of your ruleset, they are cleanly ignored.

Reply:
Got it.

Also saying that rules_swift 1.5.1 fixes a bzlmod issue, so you should update 😄.